### ISSUE TITLE
Efficientnet-lite4 model lfs pointer has been broken

### ISSUE THREAD

**Issue by tsocha** [onnx/models#363]
# Bug Report

### Which model does this pertain to?
EfficientNet-Lite4

### Describe the bug
PR #345 broke the git lfs pointer for `efficientnet-lite4.tar.gz` file
`git lfs pull` command downloads the text file instead of desired tar.gz archive.
```
$ cat vision/classification/efficientnet-lite4/model/efficientnet-lite4.tar.gz
version https://git-lfs.github.com/spec/v1
<<<<<<< HEAD
oid sha256:8eb7e6f344544c4995acac353242079a767539553a7a946ef7fdb0c3fa38b743
size 48605698
=======
oid sha256:207cc29f004ed165e2821791812d93f8713a1ce72c3264e2182fc627d52402ee
size 48621604

>>>>>>> c2f328146ba4d978ad4e90d12fddfe19ab8e8992
```
It seems that a conflict has been committed and pushed.

### Reproduction instructions

**System Information**  

OS Platform and Distribution:  Linux Ubuntu 16.04
ONNX version:  Not relevant
Backend/Runtime version:  Not relevant

Provide a code snippet to reproduce your errors.
```
git clone https://github.com/onnx/models.git
cd models
git lfs pull --include="*" --exclude="" # this one is successful
git lfs pull --include="*" --exclude="" # this one is reporting an error
```

### Notes
Error message:
```
$ git lfs pull --include="*" --exclude=""
Checkout error: pointer parsing: Expected key oid, got <<<<<<<

Errors logged to /shared/onnx/onnx-model-zoo/.git/lfs/logs/20200827T125637.20737898.log
Use `git lfs logs last` to view the log.
```

**Comment by vinitra** [comment]
Thanks for letting us know, @tsocha! @shirleysu8 -- can you take a look?

**Comment by vinitra** [comment]
#366 should be updating this.

**Comment by vinitra** [comment]
Resolved by #366
